modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -339,7 +373,5 @@ This codemod should handle most of what we've seen in code that uses Canvas Kit
   ```
 - The `<Tabs>` component no longer accepts the `currentTab` property. The Tabs uses a model now. See
   story for more details
-
-PR:
-
-- https://github.com/Workday/canvas-kit/pull/953
```

Comment:
Only removed this because it was the only PR referenced in the doc and I didn't want to do the work to map all the other PRs 😅